### PR TITLE
Include demo host app inside the project

### PR DESCRIPTION
**Pull request by axelpezzo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,12 @@
   "private": true,
   "source": "src/index.html",
   "scripts": {
+    "mock": "copyfiles -u 1 static/mockServiceWorker.js dist",
```
Comment (aditecco) [inline package.json:7]: I can't find this command in the source, the IDE errors out and the project won't start : (

```
sh: copyfiles: command not found
```

File: package.json [hunk 2]
```diff
@@ -4,11 +4,12 @@
...
     "clean": "rimraf ./dist && rimraf ./.parcel-cache",
-    "dev": "yarn run clean && parcel",
-    "build": "yarn run clean && parcel build",
+    "dev": "yarn run clean && npm run mock && parcel",
```
Comment (aditecco) [inline package.json:9]: Can we handle all the scripts with `yarn`?

```suggestion
    "dev": "yarn run clean && yarn run mock && parcel",
```

Comment (aditecco) [reply package.json:9]: To be consistent, we should also delete the package-lock in favor of yarn-lock

**Review by aditecco — CHANGES_REQUESTED** [review]

**Comment by aditecco** [comment]
@axelpezzo Ciao Alex! : ) Ho iniziato la review

File: package.json (hunk 1 above)
Comment (axelpezzo) [reply package.json:7]: I used https://www.npmjs.com/package/copyfiles, that's a global librarys and I see only now that I didn'nt add it to the dependencies. Depsite that I think that I could switch from this library to a parcel's script. I will work on that asap.

**Review by aditecco — APPROVED** [review]

**Pull request merged**